<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QWidget" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <widget class="QListWidget" name="lwAnzeige">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>531</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Bank&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>410</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Konto-Nr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ausgabe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sortieren nach</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Kundenanlage</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nahname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leKontostand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontostand</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leKontonummer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnlegen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anlegen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNachname">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leVorname">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Konto-Nr.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Überweisung</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Betrag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Von Konto-Nr.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEmpfaengerKonto">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUeberweisen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Überweisen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nach Konto-Nr.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUeberweisungsbetrag">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAbsenderKonto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Ein- und Auszahlug</string>
   </property>
   <widget class="QPushButton" name="btnEinzahlung">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Einzahlung</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Betrag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leKontonummerEinAusZahlen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAuszahlung">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auszahlung</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBetragEinAusZahlen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Konto-Nr.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leVorname</tabstop>
  <tabstop>leNachname</tabstop>
  <tabstop>leKontonummer</tabstop>
  <tabstop>leKontostand</tabstop>
  <tabstop>btnAnlegen</tabstop>
  <tabstop>leAbsenderKonto</tabstop>
  <tabstop>leEmpfaengerKonto</tabstop>
  <tabstop>leUeberweisungsbetrag</tabstop>
  <tabstop>btnUeberweisen</tabstop>
  <tabstop>leKontonummerEinAusZahlen</tabstop>
  <tabstop>leBetragEinAusZahlen</tabstop>
  <tabstop>btnEinzahlung</tabstop>
  <tabstop>btnAuszahlung</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lwAnzeige</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
